src/development/LocalTest/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM mcr.microsoft.com/dotnet/sdk:5.0-alpine AS build
+WORKDIR /LocalTest
+
+COPY ./LocalTest/LocalTest.csproj /LocalTest
+RUN dotnet restore LocalTest.csproj
+
+COPY ./LocalTest /LocalTest
+
+
+RUN ls /
+RUN dotnet build LocalTest.csproj -c Release -o /app_output
+RUN dotnet publish LocalTest.csproj -c Release -o /app_output
+
+FROM mcr.microsoft.com/dotnet/aspnet:5.0-alpine AS final
+EXPOSE 5101
+WORKDIR /app
+COPY --from=build /app_output .
+
+# Copy vaious data
+COPY ./TestData /TestData
+
+# setup the user and group (not important for LocalTest and this removes write access to /AltinnPlatformLocal)
```

Comment:
the commented lines can be deleted from the file.

Reply:
So no comment for the next guy that is about to fix a linting warning about an image configured to run as root, that /AltinnPlatformLocal needs to be handled in another way?